sats-dna/src/main/kotlin/com/sats/dna/components/card/SatsChallengeCard.kt

```diff
@@ -286,6 +284,14 @@ private fun DisabledChallengeCard(
     isDismissButtonLoading: Boolean,
     modifier: Modifier = Modifier,
 ) {
+    val dismissButtonState = rememberSatsDismissButtonState()
+
+    LaunchedEffect(isDismissButtonLoading) {
+        if (isDismissButtonLoading) {
```

Comment:
If ```isDismissButtonLoading``` becomes false again because the action failed, shouldn't ```dismissButtonState.reset()``` be invoked?

Reply:
Yeah, that's a good point! 👍